sdk/src/utils.ts

```diff
@@ -85,6 +86,28 @@ export async function sendQuery(
   }
 }
 
+export function getTx(
```

Comment:
IMO this function should be name `createTx`. The prefix `get` more implies that something already exists, you just want to gain access to it